pd/src/state.rs

```diff
@@ -11,66 +10,7 @@ pub use reader::Reader;
 pub use writer::Writer;
 
 #[instrument]
+// TODO: killme
 pub async fn new(uri: &str) -> Result<(Reader, Writer)> {
```

Comment:
The whole `state` module subtree can also be cleanly deleted, since it only did Postgres stuff, and the new `storage` module is completely separate from it.